### PR TITLE
ISSUE 230: Enable Checkstyle on the meta package

### PR DESCRIPTION
**Pull request by acoburn**
This is part of #230 and adds Checkstyle verification to the meta
package in bookkeeper-server.

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java [hunk 1]
```diff
@@ -150,7 +152,7 @@ public void processResult(int rc, String path, Object ctx) {
                     final AsyncCallback.VoidCallback stub = this;
                     scheduler.submit(new Runnable() {
                         @Override
-                        public final void run() {
+                        public void run() {
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java]: A checkstyle rule requires to remove this modifier?

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java [hunk 2]
```diff
@@ -146,21 +144,21 @@
...
      */
-    public LedgerRangeIterator getLedgerRanges();
+    LedgerRangeIterator getLedgerRanges();
 
-    /*
+    /**
      * Used to represent the Ledgers range returned from the
      * current scan.
      */
-    public static class LedgerRange {
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java]: Do not remove static

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java (hunk 1 above)
Comment (acoburn) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java]: yes, actually it does. Reverting that change results in:

    [ERROR] /Users/acoburn/Projects/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java:155:32: Redundant 'final' modifier. [RedundantModifier]

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java (hunk 2 above)
Comment (acoburn) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java]: `static` is implied in an interface. Reverting that change results in:

    [ERROR] /Users/acoburn/Projects/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerManager.java:161:5: Redundant 'static' modifier. [RedundantModifier]

**Pull request closed without merging**